libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -142,7 +142,7 @@ const AP_Param::GroupInfo AP_Arming::var_info[] = {
     // @Param: OPTIONS
     // @DisplayName: Arming options
     // @Description: Options that can be applied to change arming behaviour
-    // @Values: 0:None,1:Disable prearm display,2:Do not send status text on state change
+    // @Bitmask: 1:Disable prearm display,2:Do not send status text on state change
```

Comment:
I think we need to shift the existing options by 1.  So it should be "0:Disable prearm display" shouldn't it?

Reply:
Your right, I will update.